src/modules/item/sagas.ts

```diff
@@ -111,7 +113,11 @@ import { buildZipContents, getMetadata, groupsOf, isValidText, generateCatalystI
 import { LoginSuccessAction, LOGIN_SUCCESS } from 'modules/identity/actions'
 import { ItemPaginationData } from './reducer'
 
+export const SAVE_AND_EDIT_FILES_BATCH_SIZE = 8
+
 export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClient) {
+  const createOrEditCancelledItemsChannel = channel()
+  const createOrEditProgressChannel = channel()
```

Comment:
We have the `yield takeEvery(SAVE_MULTIPLE_ITEMS_SUCCESS, handleSaveMultipleItemsSuccess)` handler here to prompt a fetch on the items, but we don't have the same logic for the cancel one, we should add it and test it.

Reply:
excellent catch, added and tested!